apis/pkg/meta/v1alpha1/interfaces.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+var _ Pkg = &Configuration{}
+var _ Pkg = &Provider{}
+
+// Pkg is a description of a Crossplane package.
+// +k8s:deepcopy-gen=false
+type Pkg interface {
+	GetCrossplaneVersion() *string
+	GetDependencies() []Dependency
+}
+
+// GetCrossplaneVersion gets the Configuration package's Crossplane version
+// constraints.
+func (c *Configuration) GetCrossplaneVersion() *string {
+	return c.Spec.MetaSpec.Crossplane
+}
```

Comment:
It seems like we're referring to this in different places as 'crossplane', 'crossplane version', and 'crossplane constraints'. Is there a way we can be consistent? It seems like maybe `spec.crossplaneVersion` or `spec.crossplaneVersionConstraints` might make the most sense for the meta pkg if we feel okay about renaming that field (which I believe exists, but is non functional in v0.13).

Reply:
I feel like `crossplaneVersionConstraints` is most descriptive, but it is also very long :thinking:. I guess we can also have `crossplane` and then a nested `version` field...

Reply:
Yeah, `spec.crossplaneVersion` seems reasonable to me. `spec.crossplane.version` would be more future proof if we suspect we might introduce more fields under that object in future.

Reply:
Yeah I like `spec.crossplane.version` :+1: